deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -315,7 +315,11 @@ export class StackComposer {
 
         let streamingSourceType
         if (captureProxyServiceEnabled || trafficReplayerServiceEnabled || kafkaBrokerServiceEnabled) {
-            streamingSourceType = determineStreamingSourceType(kafkaBrokerServiceEnabled)
+            streamingSourceType = determineStreamingSourceType(
```

Comment:
All the parameters passed into the inner function are used in the if statement maybe remove the if statement entirely to reduce overall branching